src/utils/node-modules-transform-pattern.spec.ts

```
@@ -0,0 +1,82 @@
+  it('returns plain /node_modules/ with no exemptions', () => {
+    expect(nodeModulesTransformPattern()).toBe('/node_modules/')
+  })
+
+  describe('packageNames', () => {
+    it('exempts named extras', () => {
+      const re = new RegExp(nodeModulesTransformPattern({ packageNames: ['esm-pkg'] }))
+      expect(re.test('/repo/node_modules/esm-pkg/index.js')).toBe(false)
+      expect(re.test('/repo/node_modules/cjs-pkg/index.js')).toBe(true)
+    })
+
+  })
```

ESLint

[error] Delete `⏎`

(prettier/prettier)